docs/faq/tech-info.md

```diff
@@ -15,30 +15,33 @@ Here is a good document from the mastering plugin developer izotope comparing di
 
 ## Track Queueing
 
-MA has 3 ways of enqueuing tracks to players:
-
-**BASIC/FALLBACK**
-send tracks from the queue one by one. We just watch the player and if we see that it stopped playing at the end of a track, we tell it to play the next. This is the most basic form of enqueuing and completely handled by the MA queue controller and thus works with ALL players. This will NOT support gapless or crossfade. This setting is the default if the player has no native enqueue support or flow mode enabled. By default DLNA players use this option as well as the upcoming Home Assistant based players. Full metadata is sent to the player of what is playing.
+MA has 2 ways of enqueuing tracks to players:
 
 **NATIVE ENQUEUE SUPPORT**
-The player has native enqueue support so we tell it what the next track will be right before the current one ends. This is a special feature that not all players support. Its supported (and by default enabled) for Slimproto, Google Cast and Sonos players. Some DLNA players support it but others don't so we offer a player setting to enable it so a user can try himself if the player accepts it. benefit is full support gapless playback and in case of Slimproto and Sonos also crossfade. Also full metadata will be sent to the player what is currently playing.
+The player has native enqueue support so we tell it what the next track will be right before the current one ends. This is a special feature that not all players support. Its supported (and by default enabled) for example on Slimproto, Google Cast and Sonos players. Some DLNA players support it but others don't so we offer a player setting to enable it so a user can try himself if the player accepts it. Benefit is full support gapless playback and in case of Slimproto and Sonos also crossfade. Also full metadata will be sent to the player what is currently playing.
 
 **FLOW MODE**
-As an alternative to native enqueuing we can also send one stream of audio to the player and let Music Assistant stitch the songs together (with gapless or crossfade). This mode is used by default for Airplay and snapcast and will be used by DLNA and Cast if crossfade gets enabled. Downside of this approach is that Google cast (and most DLNA players) looses metadata display (on the speaker itself). Some DLNA players support "icy metadata" which is what we send to inform what is playing. 
+As an alternative to native enqueuing we can also send one stream of audio to the player and let Music Assistant stitch the songs together (with gapless or crossfade). This mode is used by default for Airplay, Snapcast and any mediaplayers you import from Home Assistant, and flow mode will also be used by DLNA and Cast if crossfade gets enabled. Downside of this approach is that most players loose metadata display (on the speaker itself). Some DLNA players however support "icy metadata" which is what we send to inform what is playing. 
```

Comment:
Second last sentence a typo and small grammar suggestion. Change the start of the sentence to "The downside of this approach is that most players lose metadata..."

Reply:
you can suggest the grammar/spelling by clicking this button, that way it can be adjusted on the fly:
![Uploading Scherm­afbeelding 2024-10-21 om 15.50.50.png…]()

Reply:
Sorry that pic doesn’t seem to have worked?

Reply:
![Scherm­afbeelding 2024-10-21 om 17 13 36](https://github.com/user-attachments/assets/a7837b61-8dfb-4296-a78c-bbb1238c2c6f)

Reply:
```suggestion
As an alternative to native enqueuing we can also send one stream of audio to the player and let Music Assistant stitch the songs together (with gapless or crossfade). This mode is used by default for Airplay, Snapcast and any mediaplayers you import from Home Assistant, and flow mode will also be used by DLNA and Cast if crossfade gets enabled. The downside of this approach is that most players lose metadata display (on the speaker itself). Some DLNA players however support "icy metadata" which is what we send to inform what is playing. 
```